cs_tests/cs15.c

```diff
@@ -3,7 +3,7 @@ void foo(int **w,int**x,int **y, int *z){
 	int *t;
 	t = *x;
 	*y = z;
-	*t = w;
+	*w = t;
```

Comment:
You need to add the macro (e.g. MAYALIAS or NOALIAS) to indicate what is your purpose for this change.

Reply:
Ok, since there were no macros in the initial test so I skipped adding them.
Have added now, in the latest commit.